### ISSUE TITLE
[Bug Bash] The URL “Learn about install/Uninstall Options” doesn’t have tooltips in the right panel of PM UI for non-SDK project 

### ISSUE THREAD

**Issue by CiciLi1** [NuGet/Home#13852]
### NuGet Product Used

Visual Studio Package Management UI

### Product Version

NuGet Client Dev\6.13.0.21

### Worked before?

It is not a regression since it reproes on both D16.11\34601.136 with implicit NuGet Client 5.11.6.4. 

### Impact

It bothers me. A fix would be nice

#### Notes:   

1. The repro rate is 100%.    

2. The “Help me migrate to NuGet PackageRerence” URL also doesn’t have a tooltip in the "Migrate NuGet format to PackageReference" window as below. 
![MigrateTooltips](https://github.com/user-attachments/assets/8d59cef3-cb58-4d5f-82fb-173ea37d0d86)

#### Repro Steps:    

1. Create a C# Console App (.NET framework) project.    
2. Right-click the project in Solution Explorer and select "Manage NuGet Packages…" menu item to open PM UI.    
3. Click a package from package list on Browse tab and expand “Options” node from the right panel. 

4. Hover-on the “Learn about install/Uninstall Options” URL and observe. 

#### Expected:    

The “Learn about install/Uninstall Options” URL should have tooltips. 

#### Actual:    

The “Learn about install/Uninstall Options” URL doesn’t have tooltips as below screenshot: 
![optionTooltips](https://github.com/user-attachments/assets/3e37b4da-b716-41cb-999a-b00c467080cb)
